core/triggers/interface_99_modPropalehistory_propalehistory.class.php

```diff
@@ -114,7 +114,18 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
         // Users
         $db=&$this->db;
 		//echo $action.'<br>';
-        if ($action == 'PROPAL_VALIDATE' && $conf->global->PROPALEHISTORY_AUTO_ARCHIVE) {
+        if ($action == 'PROPAL_CREATE') {
+            // only for clone proposal and if version number is used
+            if ($object->context['createfromclone'] == 'createfromclone' && !empty($object->array_options['options_propalehistory_version_num'])) {
```

Comment:
Thanks,
Perhaps your code will trigger warnings according to versions, on witch min version have you check ?
```suggestion
            if (!empty(object->context['createfromclone']) && $object->context['createfromclone'] == 'createfromclone' && !empty($object->array_options['options_propalehistory_version_num'])) {
```